docker-compose.yml

```
@@ -13,5 +13,11 @@ services:
       - NODE_ENV=development
       - CHOKIDAR_USEPOLLING=true
       - WATCHPACK_POLLING=true
+      - WEBSITE_DOMAIN=localhost
+      - DEV_PORT=3000
+      - HEALTH_CHECK_PATH=health   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)